recipes/graylog-forwarder/recipe.rb

```diff
@@ -0,0 +1,53 @@
+require_relative '../tools'
+
+class GraylogForwarder < FPM::Cookery::Recipe
+  include Tools
+
+  description 'Graylog Forwarder'
+
+  name     'graylog-forwarder'
+  version  data.version
```

Comment:
We set the `version` in `data.yml` to `3.3.9` and the `revision` to `3`. I think we wanted to use the date as revision for the package, no?

Reply:
I set revision to 3 initially. I just added the logic to the Jenkins job to set it to the date today. The Jenkins job runs a script to update the data.yml before building the package.

Reply:
Great! :+1: